content/en/Requirements/Hardware/Design_requirements_outline.adoc

```diff
@@ -0,0 +1,254 @@
+
+---
+title: "Project Tetra Hardware Requirements Outline"
+linkTitle: "Hardware Requirements Outline"
+weight: 1
+description: >
+  Summary of hardware design requirements for project tetra.
+---
+
+== Section formatting
+=== Name of requirement domain
+
+  *Goal* = Aim that the below items seek to address/provide
+  
+> Requirement or Criteria
+ 
+	- specification or test
+  
+	- *leading asterisk denotes specification or test needed
+
+	- trailing question mark denotes that definitive requirement for specification or test is unknown?
+
+
+== Overall device requirements/criteria:
+....
```

Comment:
Is there are reason for these empty block comments?

Reply:
Added for additional spacing in the final form, looked too crowded (in my eye) without.